src/molecules/Logo.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import logo from '../logo.svg';
+
+interface LogoProps {
+    height: string
+    width: string
+  }
+
+function Logo({width, height}:LogoProps) {
+    return (
+        <img src={logo} style={{width: width, height: height}} alt="logo"/>
+    )
+}
+    
+export default Logo
```

Comment:
Let's add a newline to the end of the file. You can also configure your editor to automatically add a newline at the end and trim extra new lines